Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/chan55/Chan55.java

```diff
@@ -0,0 +1,308 @@
+                    default:
+                        ext = "jpg";
+                        break;
+                }
+
+                return root.builder().s(post.board.code).s("thumb").s(arg.get("tim") + "." + ext).url();
+            }
+
+            @Override
+            public HttpUrl boards() {
+                return root.builder().s("boards.json").url();
+            }
+
+            @Override
+            public HttpUrl reply(Loadable loadable) {
+                return sys.builder().s("altpost.php").url();
+            }
+        });
+
+        setActions(new VichanActions(this) {
+            @Override
+            public void setupPost(Reply reply, MultipartHttpCall call) {
+                super.setupPost(reply, call);
+
+                if (reply.loadable.isThreadMode()) {
+                    // "thread" is already added in VichanActions.
+                    call.parameter("post", "Responder");
+                } else {
+                    call.parameter("post", "New Thread");
+                    call.parameter("page", "1");
+                }
+
+                call.parameter("json_response", "1");
+            }
+
+            @Override
+            public void handlePost(ReplyResponse replyResponse, Response response, String result) {
+                Matcher auth = Pattern.compile("dose").matcher(result);
+                Matcher banned = Pattern.compile("banned").matcher(result);
+                Matcher err = Pattern.compile("error").matcher(result);
+                Matcher postMatcher = Pattern.compile("\\.html#\\d+").matcher(result);
+                if (auth.find()) {
+                    replyResponse.requireAuthentication = true;
+                    replyResponse.errorMessage = "";
+                } else if (banned.find()) {
+                    replyResponse.errorMessage = "Você foi banido";
+                } else if (err.find()) {
+                    replyResponse.errorMessage = Jsoup.parse(result).body().text();
+                } else {
+                    Matcher m = Pattern.compile("\\d+.html").matcher(result);
+                    try {
+                        if (m.find()) {
+                            String thread = m.group(0);
+                            int dotIndex = thread.indexOf('.');
+                            replyResponse.threadNo = Integer.parseInt(thread.substring(0, dotIndex));
+
+                            if (postMatcher.find()) {
+                                String post = postMatcher.group(0);
+                                int hashIndex = post.indexOf('#');
+                                replyResponse.postNo = Integer.parseInt(post.substring(hashIndex + 1));
+                            } else {
+                                replyResponse.postNo = replyResponse.threadNo;
+                            }
+                            replyResponse.posted = true;
+                        }
+                    } catch (NumberFormatException ignored) {
```

Comment:
Only use `ignored` as a variable when you're actually doing nothing in the body of the exception. Because you're setting an error message in the response means that you're not ignoring it.